src/Nethermind/Nethermind.JsonRpc/Modules/Eth/GasPriceConfig.cs

```diff
@@ -0,0 +1,16 @@
+using Nethermind.Int256;
+
+namespace Nethermind.JsonRpc.Modules.Eth
+{
+    public static class GasPriceConfig
```

Comment:
I like it but maybe the better name will be EthGasPriceConstants or EthGasPriceEstimatorConstants?

but Constants not Config